ui/app/components/secret-engine/list.hbs

```diff
@@ -84,6 +93,9 @@
         <dd.Interactive @route={{backend.backendConfigurationLink}} @model={{backend.id}} data-test-engine-config>
           View configuration
         </dd.Interactive>
+        <dd.Interactive @route={{backend.backendConfigurationLink}} @model={{backend.id}} data-test-engine-config>
```

Comment:
We can't add this yet, until we offer support for tuning but this is where it would go.